3.3/debian-10/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -306,19 +306,21 @@ etcdctl_auth_flags() {
 ########################
 etcd_store_member_id() {
     local -a extra_flags
-
+    local member_id
```

Comment:
You need to assign it an initial value, else the script will fail with `member_id: unbound variable` on line 315.

```suggestion
    local member_id=""
```


You need to assign an initial value, else bash will report `member_id: unbound variable` on line 315.